android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -257,10 +261,56 @@ private void loadWebView() {
                 setServerBasePath(path);
             }
         }
+
+        if (!this.isMinimumWebViewInstalled()) {
+            String errorUrl = this.getErrorUrl();
+            if (errorUrl != null) {
+                webView.loadUrl(errorUrl);
+                return;
+            } else {
+                Logger.error(MINIMUM_ANDROID_WEBVIEW_ERROR);
+            }
+        }
+
         // Get to work
         webView.loadUrl(appUrl);
     }
 
+    @SuppressLint("WebViewApiAvailability")
+    private boolean isMinimumWebViewInstalled() {
+        PackageManager pm = getContext().getPackageManager();
+
+        // Check getCurrentWebViewPackage() directly if above Android 8
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            PackageInfo info = WebView.getCurrentWebViewPackage();
+            String majorVersionStr = info.versionName.split("\\.")[0];
+            int majorVersion = Integer.parseInt(majorVersionStr);
+            return majorVersion >= MINIMUM_ANDROID_WEBVIEW_VERSION;
+        }
+
+        // Otherwise manually check WebView versions
+        try {
+            PackageInfo info = pm.getPackageInfo("com.google.android.webview", 0);
```

Comment:
```suggestion
            String webViewPackage = "com.google.android.webview";
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
                webViewPackage = "com.android.chrome";
            }
            PackageInfo info = pm.getPackageInfo(webViewPackage, 0);
```

On Android N the device uses chrome as the system webview, current code returns 0 on my Android N device, but with this change it returns the correct version